bloggies_backend/src/__tests__/bookmarksRoutes.test.ts

```diff
@@ -37,44 +39,44 @@ describe("Test Post routes", function () {
                             RETURNING id`);
     validNotFavPostId = extraPostData.rows[0].id;
 
-    await db.query(`INSERT INTO favorites(post_id, user_id) VALUES (${validPostId}, ${validUserId})`);
+    await db.query(`INSERT INTO bookmarks(post_id, user_id) VALUES (${validPostId}, ${validUserId})`);
   });
 
-  /** GET /favorites/:uid => status 200, { posts }.**/
-  test("GET /favorites/:uid - retrieve favorited posts for a user by user id", async function () {
-    const resp = await request(app).get(`/favorites/${validUserId}`);
+  /** GET /bookmarks/:uid => status 200, { posts }.**/
+  test("GET /bookmarks/:uid - retrieve favorited posts for a user by user id", async function () {
+    const resp = await request(app).get(`/bookmarks/${validUserId}`);
```

Comment:
Not super critical, but the descriptions in this file can also be changed from "favorites" to "bookmarks"